readme.md

```diff
@@ -70,6 +70,8 @@ to display the generated documentation for a config.
   *  <strong>all_cities_available</strong> of user
   *
   * @_204 Successful MF!
+  * 
+  * @some_field  Description of this field from the rules method
```

Comment:
some_field does not present inside the rules method of the example class

Please also add the descpription below the 106 line